web-api/src/storage/elastic.rs

```diff
@@ -215,12 +218,42 @@ impl Client {
 
         self.query_with_bytes(url, post_data).await
     }
+
+    #[cfg(feature = "ET-3837")]
+    async fn get_score(
+        &self,
+        ids: impl IntoIterator<IntoIter = impl ExactSizeIterator<Item = &DocumentId>>,
+    ) -> Result<HashMap<DocumentId, f32>, Error> {
+        let ids = ids.into_iter();
+        if ids.len() == 0 {
+            return Ok(HashMap::new());
+        }
+
+        // https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-ids-query.html
+        let body = Some(json!({
```

Comment:
:question: I'm not sure what the score that we are getting here represents. We only ask for the score for a certain id without a target embedding.